packages/app/src/features/mainPage/components/Folder.tsx

```
@@ -0,0 +1,122 @@
+              boxShadow: "inset 0 0 0 1px var(--color-primary)",
+              backgroundColor: "var(--color-hover)",
+            }
+          : {}),
+      }}
+      aria-current={props.isActive ? "true" : undefined}
+    >
+      <div className="h-6 absolute flex items-center">
+        {!props.isRoot ? (
+          props.mode === "display" && props.childrenExists ? (
+            <span
+              role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)